### ISSUE TITLE
qtmodern style looks wrong with the newest PyQt5 version

### ISSUE THREAD

**Issue by mapfiable** [gmarull/qtmodern#52]
So I was running Python with the pyqt version that comes with Anaconda, namely [5.9.2](https://anaconda.org/anaconda/pyqt), why it comes with this old version, I don't know. I wasn't aware of it until I encountered a [bug ](https://stackoverflow.com/questions/60828957/adding-more-than-one-property-to-a-gui-built-with-qt-designer-causes-systemerror) in PyQt5, that should have been a thing of the past. Ever since I updated my pyqt version to 5.14.1 using `pip install PyQt5`, the qtmodern style is wrong, with the bigger issue being that sliders disappear when the mouse is not hovering above them and checkboxes not being visible when inside a groupbox e.g.. How can I fix this?

![qtmodern](https://user-images.githubusercontent.com/48512334/78356188-5db39c00-75af-11ea-91e7-4e63d8694ba4.jpg)

Running on Windows 10 Home build 18363.720 using:
PyCharm version: 
2019.3.4 (Professional Edition)
Anaconda vesion: 2020.02
Python version: 3.7.7
PyQt version: 5.14.1

**Comment by WBP20** [comment]
Hi, I faced same problem with scrollbars handlers, I went to the style.css file and changed colors of the vertical and horizontal handlers to "midlight" to make them visible because it has background colors (making them invisible).

**Comment by mapfiable** [comment]
Hi, thanks for your reply! Sorry for being ignorant, but which 'styles.css' file do you mean exactly?

**Comment by WBP20** [comment]
\Python\Python38-32\Lib\site-packages\qtmodern\resources\style.css

**Comment by mapfiable** [comment]
Thank you, that worked for me! I also changed the hover color to 'highlight'. It's more complicated to [change the indicator color of the checkbox](https://stackoverflow.com/q/30693600/5472354) though. Apparently, you need to re-create the [whole stylesheet](https://doc.qt.io/qt-5/stylesheet-examples.html#customizing-qcheckbox) for that, and only change what you need. But I don't know where I would find the default values.

**Comment by Jerakin** [comment]
Would love if you guys would have time to do a PR with some before/after images of your fix. With preferably the new PyQt version and the old one :)

**Comment by mapfiable** [comment]
Hey, sorry, what's a PR?

**Comment by Jerakin** [comment]
PR means [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests). Basically you can propose changes to the project to improve it, it's one of the cornerstones in Open Source applications :)

**Comment by mapfiable** [comment]
Ah, ok, I can give it a try! Not sure how that will work from a technical standpoint though..

**Comment by mapfiable** [comment]
https://github.com/gmarull/qtmodern/pull/54 gave it a try. Is that what you were asking?

**Comment by Jerakin** [comment]
Hi again!
I had a few minutes to take a look at this and it seems to have been fixed on master already (even before the PR was merged). Was the qtmodern version you where using installed with `pip install qtmodern` or did you install from master in a different way?

**Comment by mapfiable** [comment]
I used `pip install qtmodern`. Maybe the order in which I install things is relevant? I think I went Anaconda with pyqt 5.9.2 -> `pip install qtmodern` -> `pip install PyQt5`. What do you mean by it was fixed already though? If the style sheet wasn't adapted like I did, it was fixed in a different way I guess?

**Comment by Jerakin** [comment]
The project is always improving and getting bug fixes. Because of the work involved not every bugfix gets a release and there is time inbetween them (for instance, your fix have been merged but even if you run `pip install qtmodern` you will notice that your fix is not present). Since the release of `0.2.0` there have been a couple of fixes, but the `0.2.0` is the latest release distributed with `pip`.

In short `master` have bug fixes that have not yet been released. :)

**Comment by mapfiable** [comment]
Ah, I see! Thanks for the clarification.
